src/main/java/us/talabrek/ultimateskyblock/player/PlayerInfo.java

```diff
@@ -53,8 +59,8 @@ public void removeFromIsland() {
         this.hasIsland = false;
         this.setIslandLocation(null);
         this.homeLocation = null;
-        if (uuid != null) {
-            Player player = Bukkit.getPlayer(uuid);
+        if (this.uuid != null) {
```

Comment:
No need for the explicit `this.` - I know it's a matter of coding style. Mine is "less is more".